packages/api-gateway/src/utilities/errorMappers.ts

```diff
@@ -110,5 +110,3 @@ export const upsertTenantErrorMapper = (error: ApiError<ErrorCodes>): number =>
     )
     .with("certifiedAttributeAlreadyAssigned", () => HTTP_STATUS_CONFLICT)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
-
-export const emptyErrorMapper = (): number => HTTP_STATUS_INTERNAL_SERVER_ERROR;
```

Comment:
🤔  Why this ?
it's imported in [apiGatewayRouter.ts](https://github.com/pagopa/interop-be-monorepo/blob/0f2934cdb3c7dbc6e1075ad7a9516d6fca6fa60b/packages/api-gateway/src/routers/apiGatewayRouter.ts#L18).